packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -192,7 +196,32 @@ const String _iosPluginRegistryImplementationTemplate = '''//
 @end
 ''';
 
-void _writeIOSPluginRegistrant(String directory, List<Plugin> plugins) {
+const String _iosPluginRegistrantPodspecTemplate = '''
+#
+# Generated file, do not edit.
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'FlutterPluginRegistrant'
+  s.version          = '0.0.1'
+  s.summary          = 'Registers plugins with your flutter app'
+  s.description      = <<-DESC
+Depends on all your plugins, and provides a function to register them.
+                       DESC
+  s.homepage         = 'https://flutter.io'
+  s.license          = { :type => 'MIT' }
```

Comment:
MIT => BSD?

Reply:
Separate PR maybe.

Reply:
Sure - it has to be on the engine repo...